nemo/collections/tts/models/hifigan.py

```diff
@@ -76,26 +76,29 @@ def configure_optimizers(self):
             self._cfg.optim, params=itertools.chain(self.msd.parameters(), self.mpd.parameters()),
         )
 
-        self.scheduler_g = CosineAnnealing(
-            optimizer=self.optim_g,
-            max_steps=self._cfg.max_steps,
-            min_lr=self._cfg.sched.min_lr,
-            warmup_steps=self._cfg.sched.warmup_ratio * self._cfg.max_steps,
-        )  # Use warmup to delay start
-        sch1_dict = {
-            'scheduler': self.scheduler_g,
-            'interval': 'step',
-        }
-
-        self.scheduler_d = CosineAnnealing(
-            optimizer=self.optim_d, max_steps=self._cfg.max_steps, min_lr=self._cfg.sched.min_lr,
-        )
-        sch2_dict = {
-            'scheduler': self.scheduler_d,
-            'interval': 'step',
-        }
-
-        return [self.optim_g, self.optim_d], [sch1_dict, sch2_dict]
+        if hasattr(self._cfg, 'sched'):
```

Comment:
Doesn't your new _44100 yaml still have sched?

Reply:
Oh we remove it by using ~optim.sched like you suggested from command line.

Reply:
ok